templates/templates/kyma-integration.yaml

```diff
@@ -206,6 +206,50 @@ gke_upgrade_job_template_k14: &gke_upgrade_job_template_k14
             memory: 200Mi
             cpu: 80m
 
+gke_upgrade_e2e_job_template: &gke_upgrade_e2e_job_template
+  decorate: true
+  path_alias: github.com/kyma-project/kyma
+  max_concurrency: 10
+  spec:
+    containers:
+      - <<: *base_image_k15
+        securityContext:
+          privileged: true
+        command:
+          - "bash"
+        env:
+          - name: CLOUDSDK_COMPUTE_ZONE
+            value: "europe-west4-a"
+        args:
+          - "-c"
+          - "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/cluster-integration/kyma-gke-upgrade-e2e.sh"
+        resources:
+          requests:
+            memory: 200Mi
+            cpu: 80m
+
+gke_upgrade_e2e_job_template_k14: &gke_upgrade_e2e_job_template_k14
```

Comment:
Should we still support k14? Cause this job will be used from rel 1.14 which is on k8s 1.16+(at least 1.15+).